src/vos/storage_estimator/common/explorer.py

```diff
@@ -9,7 +9,7 @@
 import copy
 import sys
 
-from storage_estimator.vos_structures import VosObject, AKey, DKey, Container, Containers, VosValue, Overhead, ValType, KeyType
+from storage_estimator.vos_structures import VosObject, AKey, DKey, Container, VosValue, Overhead, ValType, KeyType
```

Comment:
(pylint-line-too-long) Line too long (115/100)
(pylint-import-error) Unable to import 'storage_estimator.vos_structures'